cpu/kw41z/vectors.c

```diff
@@ -0,0 +1,291 @@
+    dummy_handler,         /* reserved 193 */
+    dummy_handler,         /* reserved 194 */
+    dummy_handler,         /* reserved 195 */
+    dummy_handler,         /* reserved 196 */
+    dummy_handler,         /* reserved 197 */
+    dummy_handler,         /* reserved 198 */
+    dummy_handler,         /* reserved 199 */
+    dummy_handler,         /* reserved 200 */
+    dummy_handler,         /* reserved 201 */
+    dummy_handler,         /* reserved 202 */
+    dummy_handler,         /* reserved 203 */
+    dummy_handler,         /* reserved 204 */
+    dummy_handler,         /* reserved 205 */
+    dummy_handler,         /* reserved 206 */
+    dummy_handler,         /* reserved 207 */
+    dummy_handler,         /* reserved 208 */
+    dummy_handler,         /* reserved 209 */
+    dummy_handler,         /* reserved 210 */
+    dummy_handler,         /* reserved 211 */
+    dummy_handler,         /* reserved 212 */
+    dummy_handler,         /* reserved 213 */
+    dummy_handler,         /* reserved 214 */
+    dummy_handler,         /* reserved 215 */
+    dummy_handler,         /* reserved 216 */
+    dummy_handler,         /* reserved 217 */
+    dummy_handler,         /* reserved 218 */
+    dummy_handler,         /* reserved 219 */
+    dummy_handler,         /* reserved 220 */
+    dummy_handler,         /* reserved 221 */
+    dummy_handler,         /* reserved 222 */
+    dummy_handler,         /* reserved 223 */
+    dummy_handler,         /* reserved 224 */
+    dummy_handler,         /* reserved 225 */
+    dummy_handler,         /* reserved 226 */
+    dummy_handler,         /* reserved 227 */
+    dummy_handler,         /* reserved 228 */
+    dummy_handler,         /* reserved 229 */
+    dummy_handler,         /* reserved 230 */
+    dummy_handler,         /* reserved 231 */
+    dummy_handler,         /* reserved 232 */
+    dummy_handler,         /* reserved 233 */
+    dummy_handler,         /* reserved 234 */
+    dummy_handler,         /* reserved 235 */
+    dummy_handler,         /* reserved 236 */
+    dummy_handler,         /* reserved 237 */
+    dummy_handler,         /* reserved 238 */
+    dummy_handler,         /* reserved 239 */
+    dummy_handler,         /* reserved 240 */
+    dummy_handler,         /* reserved 241 */
+    dummy_handler,         /* reserved 242 */
+    dummy_handler,         /* reserved 243 */
+    dummy_handler,         /* reserved 244 */
+    dummy_handler,         /* reserved 245 */
+    dummy_handler,         /* reserved 246 */
+    dummy_handler,         /* reserved 247 */
+    dummy_handler,         /* reserved 248 */
+    dummy_handler,         /* reserved 249 */
+    dummy_handler,         /* reserved 250 */
+    dummy_handler,         /* reserved 251 */
+    dummy_handler,         /* reserved 252 */
+    dummy_handler,         /* reserved 253 */
+    dummy_handler,         /* reserved 254 */
+    dummy_handler,         /* reserved 255 */
```

Comment:
There is a check in the linker script to ensure that the interrupt vector and flash security blocks are placed at the correct location and size.